lib/index.js

```diff
@@ -129,14 +128,10 @@ function HttpProxyMiddleware(context, opts) {
     }
 
     function logError(err, req, res) {
-        var hostname = (req.headers && req.headers.host) || (req.hostname || req.host); // (websocket) || (node0.10 || node 4/5)
+        var hostname = (req.headers && req.headers.host) || (req.hostname || req.host);     // (websocket) || (node0.10 || node 4/5)
         var target = proxyOptions.target.host || proxyOptions.target;
-        var description =  errno.code[err.code] && errno.code[err.code].description;    //  description for the error code
-        
-        if (description) {
-            logger.error('[HPM] Error \'%s\' (ErrorCode=%s) occurred while trying to proxy request %s from %s to %s', description, err.code, req.url, hostname, target);
-        } else {
-            logger.error('[HPM] Error (ErrorCode=%s) occurred while trying to proxy request %s from %s to %s', err.code, req.url, hostname, target);
-        }
+        var errReference = 'https://nodejs.org/api/errors.html#errors_common_system_errors'; // link to Node Common Systems Errors page
+
+        logger.error('[HPM] Error occurred while trying to proxy request %s from %s to %s (%s) \n(%s)', req.url, hostname, target, err.code, errReference);
     }
```

Comment:
Do we really need HPM here? If this is a transitive dependency people will be confused what it means.